lib/active_model/serializer.rb

```diff
@@ -83,37 +79,6 @@ def self.type(type)
       self._type = type
     end
 
-    # @example
-    #   class AdminAuthorSerializer < ActiveModel::Serializer
-    #     attributes :id, :name, :recent_edits
-    def self.attributes(*attrs)
-      attrs = attrs.first if attrs.first.class == Array
-
-      attrs.each do |attr|
-        attribute(attr)
-      end
-    end
```

Comment:
You'll be delighted to know that I'm doing exactly that as we speak. As said, this is only half the other PR (only the association related stuff).